### DISCUSSION TITLE
API Docs URL is displayed the same even after changing it in FastAPI app.

### DISCUSSION THREAD

**Discussion by Siddhesh-Agarwal** [fastapi/fastapi-cli#97] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
from fastapi import FastAPI

app = FastAPI(
    title="test",
    docs_url="/",
    debug=True,
)

@app.get('/api/test/')
def test():
    return {'status': 'ok'}
```

### Description

So, even though the `docs_url` in the FastAPI Client is `/`, the CLI displays the docs URL as `/docs` (which is the default)
![image](https://github.com/user-attachments/assets/40b6a251-b918-4676-8d8a-ba088f93e802)

### Operating System

Windows

### Operating System Details

Windows 11

### Python Version

3.10.13

**Comment by FlavienRx** [comment]
Hey @Siddhesh-Agarwal I made this PR #119 to fix this issue 😄

**Reply by Siddhesh-Agarwal** [reply]
Yaay. Letz goo :🚀

**Comment by YuriiMotov** [comment]
There is an issue https://github.com/fastapi/fastapi-cli/issues/162.
Let's track it there
